news/_posts/2019-10-14-prestashop-1.7-modules-latest-releases-and-updates.md

```diff
@@ -0,0 +1,95 @@
+- Filters which do not match any products are now hidden ([#14655](https://github.com/PrestaShop/PrestaShop/issues/14655))
+
+A lot of bugs have been solved, including:
+
+- You can now limit filter results ([#9813](https://github.com/PrestaShop/PrestaShop/issues/9813))
+- The number of matching products after selecting filters is now working ([#14734](https://github.com/PrestaShop/PrestaShop/issues/14734))
+- Disabling the number of matching products is now working ([#9949](https://github.com/PrestaShop/PrestaShop/issues/9949))
+- Long feature titles are truncated to keep them on one line ([#9978](https://github.com/PrestaShop/PrestaShop/issues/9978))
+- Products are correctly sorted by position when you don’t show products from subcategories ([#15062](https://github.com/PrestaShop/PrestaShop/issues/15062))
+- Products are correctly sorted by price even when you have discounts ([#15223](https://github.com/PrestaShop/PrestaShop/issues/15223) & [#10010](https://github.com/PrestaShop/PrestaShop/issues/10010))
+- You can choose in which order the filters appear (#9880)
+- “Filter by” blocks are not displayed if no filters are available ([#9732](https://github.com/PrestaShop/PrestaShop/issues/9732) & [#9736](https://github.com/PrestaShop/PrestaShop/issues/9736))
+
+### Customer reassurance module (current version: 4.1.1)
+
+A major update on the module has been released last week bringing new features and design both on front office and back office. With this new version, merchants will be able to: 
+
+- Display their elements as desired on their homepage, either on the header or footer, but also on product and checkout pages  
+img
+
+- Create as many elements as desired, with the ability to add available or custom icons, title and description and choose colors  
+img
+
+- Enable or disable any of them and easily reorder them through drag and drop feature  
+img
+
+### Other bug fixes on modules 
+
+- Email subscription module (v2.5.0) - Fix on email registration issue ([#15425](https://github.com/PrestaShop/PrestaShop/issues/15425))
+- Contact form (v4.1.1) - Add a check to verify if the customer is the order’s owner
+- Shopping cart module (v2.0.2) - Fix related to add to cart from quickview issue ([#11631](https://github.com/PrestaShop/PrestaShop/issues/11631))
+- Legal compliance (v3.0.2) - Fix on label translation issue ([#9766](https://github.com/PrestaShop/PrestaShop/issues/9766))
+- Dashtrends (v2.0.2) - Fix related to incorrect calculation of sales and profit ([#9983](https://github.com/PrestaShop/PrestaShop/issues/9983))
+- Main menu (v 2.1.1) - Fix on sub menu hiding menu ([#9938](https://github.com/PrestaShop/PrestaShop/issues/9938))
+
+## And even more to come!
+
+Other new and existing modules are currently being developed or improved to bring great features to merchants and developers. They will be released step by step during the next few months. Sure hope you will enjoy them! 
```

Comment:
```suggestion
Other new and existing modules are currently being developed or improved to bring great features to merchants and developers. They will be released step by step during the next few months. We hope you will enjoy them! 
```

Reply:
who is we?
in general, please avoid the use of "we" in article when this is not clear who is we. is it the PM team? the PrestaShop company? the contributors to the modules? who else?
if it's not clear, there is always a way to write the sentence in another way, and if there no better way, then, add precision.